### PR TITLE
feat: Add link to download municipalities data

### PR DESCRIPTION
**Pull request by ptbrowne**
Add a link to download municipalities info

- Data downloaded from Sparql following a query made by Adrian
- Parsed with zod
- Formatted to csv with d3
- Data is cached for 5m

### CODE REVIEW

File: src/locales/de/messages.po [hunk 1]
```diff
@@ -270,46 +274,46 @@ msgid "histogram.municipality-count"
 msgstr "Anzahl Gemeinden"
 
 #: src/components/detail-page/price-distribution-histogram.tsx:164
-msgid "histogram.operator-count"
-msgstr "Anzahl Netzbetreiber"
+#~ msgid "histogram.operator-count"
+#~ msgstr "Anzahl Netzbetreiber"
```
Comment (bprusinowski) [inline src/locales/de/messages.po:278]: Maybe can be removed?

File: src/pages/api/municipalities-data.csv.ts [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+  const data = (await resp.json()) as unknown as SparqlResponse;
+
+  const bindings = data.results.bindings;
+  return z
+    .array(MunicipalityInfo)
+    .parse(bindings.map((mv) => mapValues(mv, (x) => x.value)));
+};
+
+const handler: NextApiHandler = async (req, res) => {
+  const period = Number(req.query.period?.toString() ?? 2024!);
```
Comment (bprusinowski) [inline src/pages/api/municipalities-data.csv.ts:138]: Nit: Probably won't happen, but maybe

```typescript
const period = Number(req.query.period?.toString() ?? new Date().getFullYear());
```

would be safer?

**Review by bprusinowski — APPROVED** [review]
LGTM! Couldn't test the behavior, as no data is fetched on the page, but code looks good 👌

File: next.config.js [hunk 3]
```diff
@@ -6,20 +6,13 @@ const withMDX = require("@next/mdx")();
...
+const { I18N_DOMAINS, WEBPACK_ASSET_PREFIX, MATOMO_ID } = process.env;
 
 const buildEnv = {
   VERSION: `v${pkg.version}`,
-  DEPLOYMENT: DEPLOYMENT,
-  CURRENT_PERIOD: CURRENT_PERIOD,
-  FIRST_PERIOD: FIRST_PERIOD,
+  DEPLOYMENT: process.env.DEPLOYMENT,
+  CURRENT_PERIOD: process.env.CURRENT_PERIOD,
+  FIRST_PERIOD: process.env.CURRENT_PERIOD,
```
Comment (adintegra) [inline next.config.js:15]: Quick question to clarify @ptbrowne: Is this referencing the correct var? The naming would imply differently.

Comment (ptbrowne) [reply next.config.js]: 😱 Thanks, I fixed it.

Comment (adintegra) [reply next.config.js]: Thanks for picking up the one in `src/env/build.js` as well 👍🏼

**Pull request merged**